Bottom: 90bb4c1456b80a158b06076838cef43bdedfc4f3
Top:    79239c355da4e30898ee1d6b5dec5786192c2e0e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 21:37:07 +0000

Refresh of board-outline-polygon-generati

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 05942ca..e758316 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1391,8 +1391,8 @@ GhidDrawMask (int side, BoxType * screen)
   poly_Free (&polygon.Clipped);
   poly_FreeContours (&polygon.NoHoles);
   /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  gui->fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  gui->graphics->fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
+//  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
+//  hid_draw_fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   hid_draw_use_mask (&ghid_graphics, HID_MASK_OFF);
